libgeopmd/src/PowercapSysfsDriver.cpp

```diff
@@ -105,7 +107,18 @@ namespace geopm
         : M_PROPERTIES{SysfsDriver::parse_properties_json(plugin_name(), powercap_sysfs_json())}
         , M_POWERCAP_RESOURCE_BY_NAME(load_powercap_resource_by_name(powercap_directory))
         , M_POWERCAP_DIRECTORY(powercap_directory)
+        , m_rollover_factor(0.0)
     {
+        try {
+            std::string factor_path = POWERCAP_DIRECTORY;
+            factor_path += "/intel-rapl/intel-rapl:0/max_energy_range_uj";
```

Comment:
Can't you get at this with ``M_PROPERTIES``, ``name_to_property_key()``, and the newly defined ``MAX_ENERGY_RANGE`` instead of duplicating the JSON data here?